### ISSUE TITLE
 IMU synthesis and training code

### ISSUE THREAD

**Issue by lambdalemon** [Xinyu-Yi/PNP#3]
Thank you for this truly amazing project!
I'm curious if the release of IMU synthesis code is still planned? How about code for training?

**Comment by Xinyu-Yi** [comment]
Yes! I will release the IMU synthesis codes later (currently part of this is in the `carticulate` repo). I'll release the codes after some reformatting.

For training codes, unfortunately, at this time we are making some technical transfer and commercializing, and as such, we are unable to share the training code. We understand these codes can be very valuable for researchers, but we really regret the inconvenience. You can try to implement the training based on the details in the paper, and if you encounter any difficulties, feel free to ask me.
